src/CBlueprintSourcemap.h

```diff
@@ -7,25 +7,22 @@
 //  Copyright (c) 2014 Apiary Inc. All rights reserved.
 //
 
-#ifndef SC_C_BLUEPRINT_SOURCEMAP_H
-#define SC_C_BLUEPRINT_SOURCEMAP_H
+#ifndef SNOWCRASH_C_BLUEPRINTSOURCEMAP_H
```

Comment:
Re  `SC_` vs `SNOWCRASH_`:  I have thought of `sc` as sort of prefix for C interface e.g. `sc_do_something()`, in C++ you would use `snowcrash` namespace eg. `snowcrash::do_something()` hence `SC_` isntead of `SNOWCRASH_`. But its OK to leave it as you have changed it.